hawkbit-rest/hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtRolloutResourceTest.java

```diff
@@ -585,6 +585,21 @@ void rolloutPagedListContainsAllRollouts() throws Exception {
                 .andExpect(jsonPath("content[1]._links.self.href", startsWith(HREF_ROLLOUT_PREFIX)));
     }
 
+    @Test
+    void retrieveRolloutWithStartAtAndForcedTimeResponseFields() throws Exception {
+        final DistributionSet dsA = testdataFactory.createDistributionSet("");
+
+        testdataFactory.createTargets(20, "target", "rollout");
+        postRollout("rollout1", 5, dsA.getId(), "id==target*", 20, Action.ActionType.FORCED, 21L, 45L);
```

Comment:
Wouldn't you use ActionType.TIMEFORCED if you specify a forcetime?

Reply:
Well typically yes, but seems that the Backend does not verify this ... So it seems I've missed this ... 
Isn't this a bug ?

Reply:
I agree, specifying action type "forced" + a forcetime could have also been mapped to a Bad Request (400). But I think it is not that bad and therefore I'm for keeping the current behavior to not break any clients.